src/esm_runscripts/filelists.py

```diff
@@ -1244,13 +1267,43 @@ def complete_one_file_movement(config, model, filetype, movement, movetype):
 
 def get_method(movement):
     if movement == "copy":
-        return shutil.copy2
+        return actually_copy
     elif movement == "link":
-        return os.symlink
+        return actually_link
     elif movement == "move":
-        return os.rename
+        return actually_move
     logger.info("Unknown file movement type, using copy (safest option).")
-    return shutil.copy2
+    return actually_copy
+
+
+def movement(func):
+    def inner(config, source_path, target_path):
+        try:
+            func(source_path, target_path)
+            return True
+        except IOError:
+            logger.error(
+                f"Could not copy {file_source} to {file_target} for unknown reasons.",
```

Comment:
The following is more explicit as an error message:
```suggestion
                f"Could not execute movement ({func.__name__}) {file_source} to {file_target} for unknown reasons.",
```